src/AspNetCoreModuleV2/CommonLib/aspnetcoreconfig.cxx

```diff
@@ -347,6 +347,19 @@ ASPNETCORE_CONFIG::Populate(
         }
     }
 
+    if (m_fWebSocketEnabled)
```

Comment:
Do we need the code before this?

Reply:
I'd prefer too. If the websocket module is disabled in the applicationhost.config, it would be ignored by removing the prior check.

Reply:
how could this happen? if websocket module was enabled while WEBSOCKET_VERSION was not set, it means Websocket failed ealrier. The websocket request should not be reached ANCM at all due to failure earlier

Reply:
Isn't websocket module the one setting variable? If it's disabled nothing would be set in theory.

Reply:
I now have more context about this one. This looks as a bad design of IIS websocket module and setup. I think we can just check server variable WEBSOCKET_VERSION as Websocket module always sets it and ignore configuration.